src/ads/ima/requester.js

```diff
@@ -406,40 +406,80 @@ Scoped.define("module:Ads.IMARequester", [
 
             /**
              * @param ad IMA Ad data
-             * @param options
+             * @param dyn
              */
-            _showCompanionAd: function(ad, options) {
+            _showCompanionAd: function(ad, dyn) {
+                var element, playerElement, position, selector, options, height, width;
+                playerElement = dyn.activeElement();
+                options = dyn.get("companion-ad");
+                if (options.split('|').length > 0) {
+                    position = options.split('|')[1] || 'bottom';
+                }
                 options = options.replace(/\].*/g, "$'").split('[');
-                var selector = options[0];
-                if (!selector) return;
-                var element = document.getElementById(selector);
+                selector = options[0];
+                if (selector) {
+                    element = document.getElementById(selector);
+                } else {
+                    element = document.createElement('div');
+                }
                 if (!element) return;
```

Comment:
This should create element as fallback when there's no object with id `selector`. 🙂
```suggestion
                if (selector) element = document.getElementById(selector);
                if (!element) element = document.createElement('div');
```